src/App.tsx

```diff
@@ -1,9 +1,14 @@
-import "./App.css";
+import './App.css';
+import { TodoList } from '@todo/components';
+import { TodoListProvider } from '@todo/store';
```

Comment:
I don't like the `imports` order, for better to put CSS (or any other styles) under all other `imports`.